tests/integration/metrics_tests/test_metrics.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+import time
+import yaml
+
+import ops
+import pytest
+import urllib3
+from pytest_operator.plugin import OpsTest
+from pathlib import Path
+from ..ha_tests import helpers as ha_helpers
+
+
+METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
+DATABASE_APP_NAME = "mongodb-k8s"
+MONGODB_EXPORTER_PORT = 9216
+MEDIAN_REELECTION_TIME = 12
+
+
+@pytest.fixture(scope="module")
+def chaos_mesh(ops_test: OpsTest) -> None:
+    ha_helpers.deploy_chaos_mesh(ops_test.model.info.name)
+
+    yield
+
+    ha_helpers.destroy_chaos_mesh(ops_test.model.info.name)
+
+
+async def get_address(ops_test: OpsTest, app_name=DATABASE_APP_NAME, unit_num=0) -> str:
```

Comment:
It looks like `app_name` is always equals to `DATABASE_APP_NAME` if so, do we really need it as a param or am I missing something?

Reply:
It is needed it for one of the HA tests, which deploys two clusters in parallel and doesn't have mixed data.

Reply:
yes, @dmitry-ratushnyy is right, we don't use `get_address` function in the `test_unique_cluster_dbs` test.

Reply:
What we should be using is the HA helper:
```
mongodb_application_name = await ha_helpers.get_application_name(ops_test, DATABASE_APP_NAME)
```
which retrieves the app name so that we can run these tests on a provided cluster